tuf/client/updater.py

```diff
@@ -665,6 +680,10 @@ def __init__(self, repository_name, repository_mirrors):
                                           'metadata_path': 'metadata',
                                           'targets_path': 'targets',
                                           'confined_target_dirs': ['']}}
+      targets_map_file:
+        The name of the targets map file. If one is not provided,
+        self.targets_map_file will be set to None and the top-level
+        targets metadata from the repository will be used.
```

Comment:
I find it confusing that the argument `targets_map_file` has the filename and the attribute `targets_map_file` has its contents. Would you mind renaming one other the other? Above in `MultiRepoUpdater` you use `targets_map_filename` for the parameter.